tests/test_ftpfs.py

```diff
@@ -166,15 +166,21 @@ def tearDown(self):
         super(TestFTPFS, self).tearDown()
 
     def test_ftp_url(self):
-        self.assertTrue(self.fs.ftp_url.startswith("ftp://127.0.0.1"))
+        self.assertTrue(self.fs.ftp_url.startswith("ftp://{}:{}@{}".format(self.user, self.pasw, self.server.host)))
```

Comment:
I guess there'd be no harm in adding in `self.server.port` here as well? And then I think this might also be able to use assertEqual instead of assertTrue?